tsconfig.json

```diff
@@ -1,6 +1,5 @@
 {
   "compilerOptions": {
-    "allowJs": true,
```

Comment:
Why did you remove `allowJs`? Some of PostGraphQL’s files like [`createPostGraphQLHttpRequestHandler.js`](https://github.com/calebmer/postgraphql/blob/6b3374baa974d8efd110970e4bd9431b8b3771f3/src/postgraphql/http/createPostGraphQLHttpRequestHandler.js) are written in plain JavaScript as it is more convenient, and we naturally play a little more loosely with types.